pkgs/applications/graphics/feh/default.nix

```diff
@@ -1,29 +1,35 @@
-{ stdenv, fetchurl, makeWrapper, xorg, imlib2, libjpeg, libpng
+{ stdenv, fetchurl, makeWrapper
+, xorg, imlib2, libjpeg, libpng
 , curl, libexif, perlPackages }:
 
+with stdenv.lib;
+
 stdenv.mkDerivation rec {
-  name = "feh-2.18.1";
+  name = "feh-${version}";
+  version = "2.18.2";
 
   src = fetchurl {
     url = "http://feh.finalrewind.org/${name}.tar.bz2";
-    sha256 = "1ck55rhh5ax1d9k9gy2crvyjwffh6028f4kxaisd8ymgbql40f2c";
+    sha256 = "09f5rfzls4h5jcrp7ylwbiljp5qzc2nbw9p2csv0pnlaixj69gil";
   };
 
   outputs = [ "out" "doc" ];
 
   nativeBuildInputs = [ makeWrapper xorg.libXt ]
-    ++ stdenv.lib.optionals doCheck [ perlPackages.TestCommand perlPackages.TestHarness ];
+    ++ optionals doCheck [ perlPackages.TestCommand perlPackages.TestHarness ];
 
   buildInputs = [ xorg.libX11 xorg.libXinerama imlib2 libjpeg libpng curl libexif ];
 
   preBuild = ''
     makeFlags="PREFIX=$out exif=1"
   '';
 
+  libPath = makeLibraryPath ([ imlib2 curl xorg.libXinerama libjpeg libpng libexif xorg.libX11 ]);
```

Comment:
I'm a bit curious why the `libPath` field is required here. Did something change in 2.18.2 to need it or was there a problem with earlier versions as well?